spec/unit/beaker/module_install_helper_spec.rb

```diff
@@ -1,12 +1,87 @@
 require 'spec_helper'
 
-describe 'ModuleInstallHelper' do
+describe Beaker::ModuleInstallHelper do
+  context 'hosts_to_install_module_on with split master/agent setup' do
+    let(:hosts) do
+      [
+        { 'roles' => %w(master database dashboard classifier) },
+        { 'roles' => ['agent'] }
+      ]
+    end
+
+    it 'returns a node with master role' do
+      expect(hosts_to_install_module_on.first['roles']).to include 'master'
+    end
+  end
+
+  context 'hosts_to_install_module_on with split master/agent setup' do
```

Comment:
Made changes to rectify this. Had to disable an rspec cop in .rubocop.yml to allow this change.